bundles/org.openhab.binding.solax/src/main/resources/OH-INF/i18n/solax.properties

```diff
@@ -62,6 +107,11 @@ channel-type.solax.last-retrieve-time-stamp.description = Last time with a succe
 channel-type.solax.raw-data-type.label = Raw Data
 channel-type.solax.raw-data-type.description = The raw JSON data retrieved from the inverter's Wi-Fi module.
 
+# add-on
+
+addon.solax.name = Solax Binding
+addon.solax.description = This is the binding for Solax inverters.
+
```

Comment:
This seem to have moved. It's usually at the beginning of the file, so I'm wondering how it got here? Did you use `mvn i18n:generate-default-translations`?

Reply:
I just cloned your repo, checked out this branch and ran above command
```shell
# theater/openhab-addons/bundles/org.openhab.binding.solax> mvn i18n:generate-default-translations
[INFO] Scanning for projects...
[INFO] 
[INFO] --------< org.openhab.addons.bundles:org.openhab.binding.solax >--------
[INFO] Building openHAB Add-ons :: Bundles :: Solax Binding 4.1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- i18n-maven-plugin:4.1.0-SNAPSHOT:generate-default-translations (default-cli) @ org.openhab.binding.solax ---
[INFO] Reading: theater\openhab-addons/bundles/org.openhab.binding.solax/src/main/resources/OH-INF/addon/addon.xml
[INFO] Reading: theater\openhab-addons/bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/channel_types.xml
[INFO] Reading: theater\openhab-addons/bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectInverter.xml
[INFO] Writing: theater\openhab-addons/bundles/org.openhab.binding.solax/src/main/resources/OH-INF/i18n/solax.properties
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  13.519 s
[INFO] Finished at: 2023-10-17T20:23:39+02:00
[INFO] ------------------------------------------------------------------------
```

and it jumped right back to the top of the file:

![image](https://github.com/openhab/openhab-addons/assets/19519842/83a853ba-b7fe-46c5-9bca-179dde67ea98)

I'm not sure what could be the reason for this problem for you, I never heard of it before. As first step, can you try running the command again?

Reply:
As another work-around, you could try:
```shell
mvn org.openhab.core.tools:i18n-maven-plugin:4.1.0-SNAPSHOT:generate-default-translations
```

Reply:
And yes, that warning seems related, but I don't know what is causing that.